NativeApp/src/components/User/View/ListItem.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, StyleSheet } from 'react-native';
 import { PropTypes as PT } from 'prop-types';
+import colours from '../../../utilities/globalStyles/theme';
+import { P } from '../../Common';
 
 const ListItem = (props) => {
```

Comment:
I would change this to a normal list style rather than a table. It's currently very crushed looking. Maybe do something like the status on the booking screen.

Reply:
cool, I have made the changes :)